build.yaml

```diff
@@ -10,12 +10,24 @@ cassandra:
 build:
   - type: maven
     version: 3.2.5
-    goals: clean compile test --fail-never -Plong
+    goals: clean compile test integration-test --fail-never -Plong
     properties: |
       com.datastax.driver.TEST_BASE_NODE_WAIT=120
       com.datastax.driver.NEW_NODE_DELAY_SECONDS=100
       cassandra.version=$CCM_CASSANDRA_VERSION
       ccm.java.home=$CCM_JAVA_HOME
+      failIfNoTests=false
+      maven.test.failure.ignore=true
+  - type: maven
```

Comment:
Not too happy about having to duplicate some of the configuration here.  The main reason is that while you can run multiple profiles in a single maven run, if goals share the same execution id only 1 is ran.  This keeps that less complex.